src/sfapi_client/_async/projects.py

```diff
@@ -5,6 +5,21 @@
     ProjectStats as ProjectBase,
 )
 from ..exceptions import SfApiError
+import pydantic
+
+
+class AsyncRole(ProjectBase):
+    client: Optional["AsyncClient"]
+
+    @pydantic.root_validator(pre=False)
+    def check(cls, values):
+        # Remove keys that are None
+        items = list(values.items())
+        for key, val in items:
+            if val is None:
+                del values[key]
```

Comment:
Not sure what this is doing? The fields all default to `None`, so removing the keys that are `None` will have no effect? What am I missing?

Reply:
My thought was to distinguish between the project which should have these filled and the roles which never have them filled since they come from the same base model. If it doesn't make sense and we just leave them as `None` I can it out.

Reply:
If they are never set, should we not have role inherit from that base class?

Reply:
Looking into it more it may be an error with the model from the openapi output. The example model from the `GET /account/roles` is the same as `GET /account/projects` in the swagger UI but `roles` only returns a subset of that when you try it out.

Reply:
May be an error in the annotations used to generate the open api spec ...

Reply:
I will take a look at the models in the API

Reply:
Looks like they need to provide a separate model for roles, I will see if I can put together a PR.